oaknational/Oak-Web-Application
# fix: mobile curriculum image size

## Description

- Swaps out 'Image' for OakImage (which uses the next future image) on curriculum page.
- This was the only image on the site that still used the next/image and could be causing the resizing mobile bug issue, if not good for consistency anyway .

## Issue(s)

Fixes #

## How to test

1. Go to develop your curriculum (hexagonal image) 
2. Click on _______
3. You should see _______

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
